File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int res = 0;
        for (auto &str:details) {
            if (stoi(str.substr(11, 2)) > 60)
                ++res;
        }
        
        return res;
    }
};

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.rs
impl Solution {
    pub fn count_seniors(details: Vec<String>) -> i32 {
        let mut res = 0;
        
        for str in details {
            let age:i32 = str[11..13].parse::<i32>().unwrap();
            if age > 60 {
                res += 1;
            }
        }
        
        res
    }
}